accessibility-101.md

```diff
@@ -0,0 +1,196 @@
+
+###Ok, so accessibility is a big deal. Now how do we implement  it?
+
+That question, sadly, is harder to answer than it may seem. The Harry Potter quote at the top is there for a reason, and its not my being an avid Fantasy reader. :)
+
+As I stated above, accessibility is important for a large group of different people, each with their own needs. Making your website work for literally everyone is a large, on-going task.
+
+To bring a bit of a method to the madness, the Web Content Accessibility Guidelines or [WCAG](https://www.wuhcag.com/web-content-accessibility-guidelines/) were composed. This document contains a number of criteria you can use to check your website.
+For now, I will cover some of the most important basics here. I will point you at the low-hanging fruits, so to speak. In subsequent articles, I will discuss more advanced techniques like [WAI-ARIA]  which is important for JavaScript-based apps.
+
+####Talk like the natives
+The HTML specification is a document that describes how the language should be used to build websites. Assistive technologies, like screenreaders, speech recognition programs etc. are aware of this document. Web developers however, often are not, or at least not enough, and think something like this is ok:
+
+```
+<div class="awesome-button"</div>
+
+<span><strong>Huge heading I will style with CSS later</strong></span>
+
+<span class="clickable-with-JavaScript">English</span>
+```
+
+ Guess what? All three of these elements break several criteria of WCAG and therefore are not accessible at all. 
+ 
+ The first element breaks the so-called 'name, role, value'-criterium, which states that all elements on a web page should expose their name, their role (like button) and their value (like the contents of an edit field) to assistive technologies. This div actually doesn't provide any of the three, rendering it invisible to screenreaders.
+ 
+ The second element looks like a heading visually after styling it with CSS, but semantically is a span. Therefore, assistive technologies won't know its a heading. A screenreader will read this as regular text, instead of a heading. Screenreaders often have a hotkey to quickly jump to the nearest heading, this heading will not be included in that scope.
+ 
+ The third element could for example be an element a user can click to change the language of the website. Maybe a fancy animated menu of languages will expand when it is clicked. However, this is also a span and does not expose its role (link, or button), making assistive technologies think this is just the word English with some styling.
+ 
+ Spans and divs are non-elements. They are meant to contain other elements, not to be elements themselves. You can fix these in two ways:
+ 
+- You can manually add ARIA-attributes to the elements above. This is an advanced topic and outside the scope of this article.
+- Or, you can simply do this:
+ 
+ ``
```

Comment:
You only have two back ticks instead of three. Also you should use html for syntax highlight.